File: WebAPI/DataAccessLayer/Models/Customer.cs
using System.Collections.Generic;

namespace DataAccessLayer.Models
{
    public partial class Customer
    {
        public Customer()
        {
            Order = new HashSet<Order>();
        }

        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string Email { get; set; }
        public string CustomerAddress { get; set; }
        public int? Phone { get; set; }
        public byte[] Password { get; set; }
        public byte[] Salt { get; set; }

        public virtual ICollection<Order> Order { get; set; }
    }
}

File: shopping-cart/src/app/models/customer.ts
export class Customer {

    customerId : number;
    email: string;
    customerName: string;
    customerAddress: string;
    phone : number;

    constructor(customerId, email, customerName, customerAddress, phone) {
        this.customerId = customerId;
        this.email = email;
        this.customerName = customerName;
        this.customerAddress = customerAddress;
        this.phone = phone;
    }
}